.github/workflows/app_check.yml

```diff
@@ -70,3 +70,32 @@ jobs:
     - name: PodLibLint FirebaseAppCheck Cron
       # TODO: Remove --allow-warnings when stabilized.
       run: scripts/third_party/travis/retry.sh scripts/pod_lib_lint.rb FirebaseAppCheck.podspec --platforms=ios ${{ matrix.flags }}
+
+  spm:
+    # Don't run on private repo unless it is a PR.
+    if: (github.repository == 'Firebase/firebase-ios-sdk' && github.event_name == 'schedule') || github.event_name == 'pull_request'
+    runs-on: macOS-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Initialize xcodebuild
+      run: scripts/setup_spm_tests.sh
+    - name: iOS Unit Tests
+      run: scripts/third_party/travis/retry.sh ./scripts/build.sh FirebaseAppCheckUnit iOS spm
+    - name: iOS Swift Unit Tests
+      run: scripts/third_party/travis/retry.sh ./scripts/build.sh FirebaseAppCheckSwift iOS spm
+
+  spm-cron:
+    # Don't run on private repo.
+#    if: github.event_name == 'schedule' && github.repository == 'Firebase/firebase-ios-sdk'
+    runs-on: macOS-latest
+    strategy:
+      matrix:
+        target: [tvOS, macOS, catalyst]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Initialize xcodebuild
+      run: scripts/setup_spm_tests.sh
+    - name: Unit Tests
+      run: scripts/third_party/travis/retry.sh ./scripts/build.sh FirebaseAppCheckUnit ${{ matrix.target }} spm
+    - name: Swift Unit Tests
+      run: scripts/third_party/travis/retry.sh ./scripts/build.sh FirebaseAppCheckSwift ${{ matrix.target }} spm
```

Comment:
`FirebaseAppCheckUnit` should be `AppCheckUnit` and `FirebaseAppCheckSwift` should be `AppCheckUnitSwift` based on the target name. Alternatively you may add schemes with different names or rename the test targets.

Reply:
🤦  thanks!